automation/project/Dependencies.scala

```diff
@@ -15,7 +15,7 @@ object Dependencies {
   val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
   val excludeWorkbenchGoogle = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-google" + scalaV)
 
-  val workbenchServiceTestV = "0.17-bb34724"
+  val workbenchServiceTestV = "0.19-84b9dfa2-SNAP"
```

Comment:
This will change after https://github.com/broadinstitute/workbench-libs/pull/730/files merges